api/cas-server-core-api-configuration-model/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -149,6 +149,15 @@
         "reason": "management endpoints security under cas.monitor.endpoints.endpoint.*. e.g. cas.monitor.endpoints.endpoint.defaults.access[0].",
         "level": "error"
       }
+    },
+    {
+      "name": "cas.authn.pac4j.saml[0].signLogoutRequests",
+      "type": "java.lang.Boolean",
+      "description": "Whether logout requests and responses should be signed.",
+      "deprecation": {
+        "reason": "It is a duplication of signServiceProviderLogoutRequest.",
```

Comment:
Specify the full path to the replacement setting, and please use the same format when naming the setting.